File: Day 1/1 Find Characters in String/solution.py
def findCharcterInString(word, character):
  for index, char in enumerate(word):
    if char == character:
      return index
  return -1

def findAllCharcterIndexesInString(word, character):
  indexes = []
  for index, char in enumerate(word):
    if char == character:
      indexes.append(index)
  return indexes

File: Day 1/1 Find Characters in String/solution.cs
 public static int FindCharcterInString(string word, char character)
        {

            for (int index = 0; index < word.Length; index++)
            {
                
                if (word[index] == character)
                {

                    return index;
                }


            }

            return -1;
        }

        public static List<int> FindAllCharcterIndexesInString(string word, char character)
        {
            List<int> ints = new List<int>();

            for (int index = 0; index < word.Length; index++)
            {


                if (word[index] == character)
                {

                    ints.Add(index);
                }


            }

            return ints;
        }